src/lib/bootloader/sysconfig.rb

```diff
@@ -120,14 +120,12 @@ def ensure_file_exists_in_target
       return if File.exist?(File.join(destdir, "/etc/sysconfig"))
 
       Yast::WFM.Execute(Yast::Path.new(".local.mkdir"),
-        File.join(destdir, "/etc/sysconfig")
-                       )
+        File.join(destdir, "/etc/sysconfig"))
       Yast::WFM.Execute(Yast::Path.new(".local.bash"),
-        "touch #{destdir}/etc/sysconfig/bootloader"
-                       )
+        "touch #{destdir}/etc/sysconfig/bootloader")
     end
 
-    def temporary_target_agent(&block)
```

Comment:
Well, this is is a bit controversial change. Rubocop says that `yield` is faster, but for me the `&block` style is better readable.

You can see just from the method header that it takes a block. After the change this is hidden in the method body  and without any documentation comment it's even worse... :worried: 

Here I'd prefer a different default for YaST. What do you think about it?